tests/test_rpc.py

```diff
@@ -16,6 +16,7 @@
     get_txid,
     spend_coins,
     sign_and_broadcast,
+    sign_and_broadcast_psbt,
```

Comment:
Thanks for cleaning up my mess. :)

There is already a `sign_and_broadcast` as you can see so maybe we should avoid duplicating them. But not here. Let's not scope creep this PR.